src/v0/destinations/customerio/util.js

```diff
@@ -103,8 +103,9 @@ const isdeviceRelatedEventName = (eventName, destination) =>
 const identifyResponseBuilder = (userId, message) => {
   const rawPayload = {};
   // if userId is not there simply drop the payload
-  if (!userId) {
-    throw new InstrumentationError('userId not present');
+  const id = userId || getFieldValueFromMessage(message, "email");
+  if (!isDefinedAndNotNull(id)) {
```

Comment:
Is `isDefinedAndNotNull` utility required here? We can achieve the same result with the native JS. 
`if (!userId) will check for false values (null, undefined, 0, an empty string '', and NaN)`

Reply:
There are many issues coming up in customerIO so we want to make sure nothing comes up now. Hence I have included this.
Also for what case is left for which we use `isDefinedAndNotNull` then?

Reply:
isDefinedAndNotNull will generally use for object.

Reply:
isDefinedAndNotNull is using `loadash`. Since native JS will give same result on `userId`. 
Why are we using the separate package?